### PR TITLE
crab fixed

### LINKED ISSUE #26: No working crab config exists

### PR DESCRIPTION
**Pull request by YujiLee301**
Hi Ram,

I have taken efforts and made everything works in crabjobs based on our master branch.

Please change your storage location of T2 site in crab_cfg.py and run command:
crab submit --config=crab_cfg.py

Looking for your review, thanks in advance.

Best,
Yuji

### CODE REVIEW

File: FrameworkJobReport.xml
Comment (ram1123) [inline FrameworkJobReport.xml:1]: Delete this file.

File: crab_script.py
Comment (ram1123) [inline crab_script.py:1]: Do we need this file? I don't see importing this module or mention in the README. Please check

**Review by ram1123 — CHANGES_REQUESTED** [review]
Delete directory `crab_projects_sendPythonFolderFalse`. This is created when we submit the job.

File: FrameworkJobReport.xml
Comment (YujiLee301) [reply FrameworkJobReport.xml:1]: I tested without this file, and the submission can not be successful.

File: crab_script.py
Comment (YujiLee301) [reply crab_script.py:1]: No, we do not need it. Actually post_proc.py is what makes sense. So I deleted this module.

File: FrameworkJobReport.xml
Comment (ram1123) [reply FrameworkJobReport.xml:1]: hmmm, strange. This job report is created when we run it locally, by switching the job report boolean.

This report helps us to set the crab job parameters like the number of jobs and how many events are in each job.

How does the crab job depend on this file?

**Comment by ram1123** [comment]
Hi @YujiLee301 ,

It looks like you do not stat from the "HZZ_Analysis" branch for the crab job update.

Remember that our master branch is "HZZ_Analysis". So, for any update start from the "HZZ_Analysis" branch as a base branch then rename it (say `crabjobs`).

As you didn't start from the "HZZ_Analysis" branch there is a merge conflict as I updated the base branch of the job. Please check and fix it.

**Comment by ram1123** [comment]
This update moved to PR #31

**Pull request closed without merging**